bin/install

```diff
@@ -58,19 +58,16 @@ omf_install() {
     die "Could not clone the repository → ${OMF_PATH}:${OMF_REPO_BRANCH}"
   fi
 
-  pushd ${OMF_PATH} >/dev/null 2>&1
-
   local git_rev=$(git rev-parse HEAD) >/dev/null 2>&1
   local git_upstream=$(git config remote.upstream.url)
 
   if [ -z "${git_upstream}" ]; then
-    git remote add upstream ${git_uri}
+    git -C ${OMF_PATH} remote add upstream ${git_uri}
```

Comment:
Mmmh the first release of git with this option is [this commit](https://github.com/git/git/commit/44e1e4d67d5148c245db362cc48c3cc6c2ec82ca) which is the version 1.8.5, which dates back from 27 Nov. 2013.

If this isn't enough I can use the option `--git-dir` and `--work-tree` which are both present since [this commit](https://github.com/git/git/commit/892c41b98ae2e6baf3aa13901cb10db9ac67d2f3), which is the version 1.5.3 released the 07 June 2007.